cmd/entrypoint/runner.go

```diff
@@ -58,6 +59,31 @@ func (rr *realRunner) Run(ctx context.Context, args ...string) error {
 	// main process and all children
 	cmd.SysProcAttr = &syscall.SysProcAttr{Setpgid: true}
 
+	if os.Getenv("TEKTON_RESOURCE_NAME") == "" && os.Getenv(pod.HermeticEnvVar) == "1" {
+		cmd.SysProcAttr.Cloneflags = syscall.CLONE_NEWNS |
```

Comment:
I think you can call the "dropNetworking" function instead - I actually did commit that part!

https://github.com/tektoncd/pipeline/blob/1f5980f8c8a05b106687cfa3e5b3193c213cb66e/cmd/entrypoint/namespaces_linux.go#L25

Reply:
oooh ty!